### PR TITLE
more fathead metadata

### PR DESCRIPTION
**Pull request by jdorweiler**
Added a few more fields to the fathead source options and switched skip_end to use a text box.
@MariagraziaAlastra @russellholt

### CODE REVIEW

File: src/templates/commit_page.handlebars [hunk 1]
```diff
@@ -368,6 +362,18 @@
                             </span>
                         </li>
                         <li>
+                            <i class="section-group__item icon-check{{#is_false original.src_options.is_fanon}}-empty{{/is_false}}" data-field="is_fanon"/>
+                            <span class="check-txt">
+                                Is Fanon
```
Comment (russellholt) [inline src/templates/commit_page.handlebars]: what's this mean?

Comment (jdorweiler) [inline src/templates/commit_page.handlebars]: Sounds like these are mediawiki source that can sometimes have articles on real-world topics.  We want to be careful about showing them.  I didn't pick the name :smile:

Comment (russellholt) [inline src/templates/commit_page.handlebars]: Cool

**Comment by MariagraziaAlastra** [comment]
LGTM!

File: src/templates/commit_page.handlebars (hunk 1 above)
Comment (mrshu) [inline src/templates/commit_page.handlebars]: @russellholt Sorry for the late reply, @jdorweiler did describe what it really is. Pretty much every bigger wikia has that.

**Pull request merged**